MultiplayerMod/Network/IMultiplayerServer.cs

```diff
@@ -17,6 +17,7 @@ public interface IMultiplayerServer {
     void Send(IMultiplayerCommand command, MultiplayerCommandOptions options = MultiplayerCommandOptions.None);
 
     event Action<ServerStateChangedEventArgs> StateChanged;
-    event Action<PlayerConnectedEventArgs> PlayerConnected;
+    event Action<PlayerEventArgs> PlayerConnected;
+    event Action<PlayerEventArgs> PlayerDisconnected;
```

Comment:
Just `<IPlayer>`? Since we're removing unnecessary args.

Reply:
Done